waves-ext/src/main/scala/com/wavesplatform/dex/grpc/integration/services/WavesBlockchainApiGrpcService.scala

```diff
@@ -180,6 +189,14 @@ class WavesBlockchainApiGrpcService(context: ExtensionContext)(implicit sc: Sche
           CheckedBroadcastResponse(CheckedBroadcastResponse.Result.Failed(CheckedBroadcastResponse.Failure(message, canRetry = false)))
       }
 
+  private def handleTxInUtx(tx: exchange.ExchangeTransaction): Future[CheckedBroadcastResponse.Result] =
+    context.transactionsApi.broadcastTransaction(tx).map { //todo: check with tests
```

Comment:
Actually, we have the `ReBroadcastUntilConfirmedTestSuite` that does this thing

Reply:
Fixed